.golangci.yml

```diff
@@ -5,52 +5,73 @@ run:
   skip-dirs:
     - pkg/pb
 linters:
-  enable:
-    - asciicheck
-    - bidichk
-    - bodyclose
-    - containedctx
-    - contextcheck
-    - decorder
-    - durationcheck
-    - errcheck
-    - errchkjson
-    - errname
-    - errorlint
-    - execinquery
-    - exhaustive
-    - exportloopref
-    - gocognit
-    - goconst
-    - gocritic
-    - gocyclo
-    - godot
-    - gofumpt
-    - gomnd
-    - goprintffuncname
-    - gosec
-    - gosimple
-    - govet
-    - importas
-    - ineffassign
-    - lll
-    - makezero
-    - misspell
-    - nakedret
-    - nestif
-    - nilerr
-    - noctx
-    - nolintlint
-    - paralleltest
-    - prealloc
-    - staticcheck
-    - typecheck
-    - unconvert
-    - unparam
-    - unused
-    - wrapcheck
-    - wsl
-    - importas
+  enable-all: true
+  disable:
+   - asasalint
+   - cyclop
+   - deadcode
+   - depguard
+   - dogsled
+   - dupl
+   - dupword
+   - exhaustivestruct
+   - exhaustruct
+   - forbidigo
+   - forcetypeassert
+   - funlen
+   - gci
+   - ginkgolinter
+   - goconst
+   - gocheckcompilerdirectives
+   - gochecknoglobals
+   - gochecknoinits
+   - godox
+   - goerr113
+   - gofmt
```

Comment:
might be useful

Reply:
@VladislavsPerkanuks, it is already covered by `gofumpt`.